gdal/frmts/jpegxr/frmt_jpegxr.html

```diff
@@ -0,0 +1,80 @@
+<html>
+<head>
+<title>JPEGXR -- JPEG XR File Format</title>
+</head>
+
+<body bgcolor="#ffffff">
+
+<h1>JPEGXR -- JPEG XR File Format</h1>
+
+<p>The JPEG XR format is supported for reading, and batch writing, but not
```

Comment:
@schwehr It's copy of [frmts/jpeg/frmt_jpeg.html:10](https://github.com/OSGeo/gdal/blob/fc71c9e61c68a90972c01b3e1c97208fd789b03b/gdal/frmts/jpeg/frmt_jpeg.html#L10). Since, JPEG driver is well-established in GDAL, I used it as an examplar and followed the wording.